tools/migrate.py

```diff
@@ -0,0 +1,76 @@
+import json
+from typing import Optional
+
+import polars as pl
+import typer
+
+from luxonis_ml.data import LuxonisDataset
+from luxonis_ml.data.utils.constants import LDF_VERSION
+from luxonis_ml.utils import environ
+
+
+def migrate_dataframe(df: pl.DataFrame) -> pl.DataFrame:
+    # Version 2.0
+    if "class_name" in df.columns:
+        return df
+
+    return (
+        df.rename({"class": "class_name"})
+        .with_columns(
+            [
+                pl.col("task").alias("task_type"),
+                pl.col("task").alias("task_name"),
+                pl.lit("image").alias("source_name"),
+            ]
+        )
+        .select(
+            [
+                "file",
+                "source_name",
+                "task_name",
+                "created_at",
+                "class_name",
+                "instance_id",
+                "task_type",
+                "annotation",
+                "uuid",
+            ]
+        )
+    )
+
+
+def main(
+    dataset_name: Optional[str] = None, team_id: Optional[str] = None
+) -> None:
+    team_id = team_id or environ.LUXONISML_TEAM_ID
+    if dataset_name is None:
+        datasets = LuxonisDataset.list_datasets(
+            team_id, list_incompatible=True
+        )
+    else:
+        datasets = [dataset_name]
+
+    for dataset_name in datasets:
+        base_path = (
+            environ.LUXONISML_BASE_PATH
+            / "data"
+            / team_id
+            / "datasets"
+            / dataset_name
+        )
+        for parquet_file in (base_path / "annotations").glob("*.parquet"):
```

Comment:
Can we check LDF_VERSION of the dataset and migrate with this migrate function only if previous one is e.g. 1.0? Just so we "future proof" this tool so when e.g. LDF 3.0 goes out we'll probably have different migrate function from 1.0->3.0 and different from 1.0->2.0 or 2.0->3.0

Reply:
Yea this is more a suggestion, not needed at this moment.